foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -12,10 +12,10 @@ Some of the student solutions below contain buttons, text, and other elements. T
 
 Since this is the first JavaScript project being built from scratch, it's important to remember the wise words from the [Problem Solving lesson](https://www.theodinproject.com/lessons/foundations-problem-solving). For each step in this project, be sure to do the following
 
-1. Plan or pseudocode your solution.
-1. Write the code.
-1. Test your code to make sure it works.
-
+1. Understand the problem
+2. Plan the solution.
+3. Pseudocode the solution.
+4. Divide & conquer (including coding the solution)
```

Comment:
Small nit about punctuation, as well as following our [style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md) where we need to use `1.` lazy numbering for ordered lists, and ensure they're surrounded by a single blank line.
```suggestion
1. Understand the problem.
1. Plan the solution.
1. Pseudocode the solution.
1. Divide & conquer (including coding the solution).

```

Reply:
@CodeCara just a courtesy check in if you've seen the above change requested.